hub/policy/src/main/java/uk/gov/ida/hub/policy/logging/HubEventLogger.java

```diff
@@ -75,12 +76,17 @@
     private final EventSinkProxy eventSinkProxy;
     private final ServiceInfoConfiguration serviceInfo;
     private final EventEmitter eventEmitter;
+    private final boolean sendToRecordingSystem;
 
     @Inject
-    public HubEventLogger(ServiceInfoConfiguration serviceInfo, EventSinkProxy eventSinkProxy, EventEmitter eventEmitter) {
+    public HubEventLogger(ServiceInfoConfiguration serviceInfo,
+                          EventSinkProxy eventSinkProxy,
+                          EventEmitter eventEmitter,
+                          PolicyConfiguration configuration) {
```

Comment:
Pulling in the whole configuration just for a single boolean feels slightly overkill. Is there a nice way to only pass in the things that we care about?